### PR TITLE
Increment the Dataflow runner major version to 7.

### PR DESCRIPTION
**Pull request by mdvorsky**
R: @herohde

### CODE REVIEW

File: runners/google-cloud-dataflow-java/build.gradle [hunk 1]
```diff
@@ -34,7 +34,7 @@ evaluationDependsOn(":model:fn-execution")
 
 processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
-    'dataflow.legacy_environment_major_version' : '6',
+    'dataflow.legacy_environment_major_version' : '7',
     'dataflow.fnapi_environment_major_version' : '1',
```
Comment (herohde) [inline runners/google-cloud-dataflow-java/build.gradle]: Please also update the fnapi version to 7 throughout.

**Comment by herohde** [comment]
LGTM otherwise

File: runners/google-cloud-dataflow-java/build.gradle (hunk 1 above)
Comment (mdvorsky) [reply runners/google-cloud-dataflow-java/build.gradle]: Thanks Henning. I'd prefer to change the fniapi separately, as that's an orthogonal change. Moreover, I'm not even sure it's safe (Does fnapi support major versions all the way up to 7?)

Comment (herohde) [reply runners/google-cloud-dataflow-java/build.gradle]: Dataflow FnAPI supports major version 7, but it's fine to change separately.

**Comment by mdvorsky** [comment]
Hi @lukecwik, would you mind taking a look?

**Comment by lukecwik** [comment]
Run Dataflow ValidatesRunner

**Comment by lukecwik** [comment]
Run Python Dataflow ValidatesRunner

**Comment by lukecwik** [comment]
Change looks fine, waiting on postcommits for validation.

**Pull request merged**